config/config.go

```diff
@@ -29,6 +29,10 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	UpstreamBundlesURL = "https://github.com/clearlinux/clr-bundles/archive/"
+)
+
```

Comment:
Default values should be set inside `LoadDefaultsForPath` as all the others. This const is not needed. Se my other comments for further explanation.